lib/utils/express.ts

```diff
@@ -0,0 +1,38 @@
+import * as express from "express";
+import * as helmet from "helmet";
+import * as csp from "helmet-csp";
+import * as referrerPolicy from "referrer-policy";
+
+export function getSecureExpressApp(): express.Express {
+  const app = express();
```

Comment:
I would keep the responsibility of creating an express app outside of this function and instead have this function just configure the app with the security plugins

Reply:
ok